docs/DeveloperGuide.md

```diff
@@ -209,7 +209,58 @@ _to be updated_
 ---
 
 ### Schedule-related features
-_to be updated_
+![ScheduleUML](uml/classDiagrams/images/scheduleComponent.png)
+
+User are able to create and make changes to a 7-day workout plan schedule using the WerkIt application. For each day, user are only allowed
```

Comment:
Perhaps the word 'User' can be changed to its plural form 'Users'?

Reply:
Thanks for pointing that out!